README.md

```diff
@@ -41,7 +41,7 @@ A deployment guide is available from the Fortinet Document Library:
 ## Project development history
 | Version | Details | Documentation |
 | ------- | ------- | ------------- |
-| 3.0 (latest) | The AWS portion of this project has been moved to [fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws). Going forward, this project will be maintained for Azure only. | [ FortiGate / FortiOS 6.2 / Deploying auto scaling on Azure](https://docs.fortinet.com/vm/azure/fortigate/6.2/azure-cookbook/6.2.0/161167/deploying-auto-scaling-on-azure) |
+| 3.0 (latest) | The AWS portion of this project has been moved to [fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws). Going forward, this project will be maintained for Azure only. | [ FortiGate / FortiOS 6.4 / Deploying auto scaling on Azure](https://docs2.fortinet.com/vm/azure/fortigate/6.4/azure-cookbook/6.4.0/161167/deploying-auto-scaling-on-azure) |
```

Comment:
We haven't officially supported FOS 6.4 in this project. It can be left unchanged.

Reply:
Agreed. See my previous comment.